source/SIL.AppBuilder.Portal.Frontend/src/redux-store/provider.tsx

```diff
@@ -68,7 +68,12 @@ export default class ReduxProvider extends React.Component<IProps> {
       //       we need to manually set this specific key
       setCurrentOrganizationId(currentState.data.currentOrganizationId);
 
-      localStorage.setItem('reduxState', JSON.stringify(toPersist));
+      try {
+        localStorage.setItem('reduxState', JSON.stringify(toPersist));
+      } catch (e) {
+        console.error(toPersist, e);
```

Comment:
if we ever try to to persist a graph cycle, we'll get an error. but we need to know what the data looked like before the stringify